### PR TITLE
2021-12-09 release

### PR DESCRIPTION
**Pull request by palma21**

### CODE REVIEW

**Review by andyzhangx — APPROVED** [review]
/lgtm

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,34 @@
...
+* Konnectivity rollout has been halted for the rest of the year. We will continue the rollout in the new calendar year.
+* AKS is implementing auto-cert rotation slowly over the next few months. We have already enabled the following regions westcentralus, uksouth, eastus, australiacentral, and australiaest. If you have clusters in those regions please run a cluster upgrade in order to have that cluster configured for auto-cert rotation. The following regions brazilsouth, canadacentral, centralindia, and eastasia will be released in January after the holidays as the next group of regions. We will update the release notes will the upcoming schedule going forward until all regions are deployed.
+* Kubernetes 1.19 will be removed on 2021-01-31.
+* Starting with 1.23 AKS will follow upstream kubernetes and deprecate in-tree azure authentication which is marked for deprecation to be replaced with 'exec'. If you are using Azure CLI or Azure clients, AKS will download kubelogin for users automatically. If outside of Azure CLI, users need to download and install kubelogin in order to continue to use kubectl with AAD authentication. <https://github.com/Azure/kubelogin>
```
Comment (Kaarthis) [inline CHANGELOG.md:13]: upstream kubernetes and deprecate in-tree azure authentication which is marked for deprecation to be replaced with 'exec' can we simplify this sentence on deprecation - it comes out a lil confusing.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,34 @@
...
+* Features
+  * Kubernetes 1.22 is now GA.
+  * New Kubernetes patch versions released, 1.20.13, 1.21.7, 1.22.4.
+* Preview Features
+  * AKS GitOps agent extension is now in Public Preview.
+  * Microsoft Defender for containers is now in Public Preview.
+* Bug Fixes
+  * Corrected validation that silently ignored updates to HTTP proxy settings.
+  * Fixed issue that blocked creation of 0 node nodepools.
+  * CSI driver probe timeout increased to avoid driver crashes on small VM sizes.
```
Comment (Kaarthis) [inline CHANGELOG.md]: Should we call out what was the increase ?

File: CHANGELOG.md (hunk 1 above)
Comment (palma21) [reply CHANGELOG.md:13]: @miwithro

File: CHANGELOG.md (hunk 2 above)
Comment (palma21) [reply CHANGELOG.md]: Added

Comment (palma21) [reply CHANGELOG.md]: fwiw this is not the same as resources, since this doesn't affect scheduling

File: CHANGELOG.md (hunk 1 above)
Comment (miwithro) [reply CHANGELOG.md:13]: Looks good from my end.  Thank you.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,34 @@
...
+
+### Announcement
+
+* From Kubernetes 1.23, containerD will be the default container runtime for Windows node pools. Docker support will be deprecated in Kubernetes 1.24. You are advised to test your workloads before Docker deprecation happens by following the documentation here https://docs.microsoft.com/en-us/azure/aks/windows-container-cli#add-a-windows-server-node-pool-with-containerd-preview.
+* Konnectivity rollout has been halted for the rest of the year. We will continue the rollout in the new calendar year.
+* AKS is implementing auto-cert rotation slowly over the next few months. We have already enabled the following regions westcentralus, uksouth, eastus, australiacentral, and australiaest. If you have clusters in those regions please run a cluster upgrade in order to have that cluster configured for auto-cert rotation. The following regions brazilsouth, canadacentral, centralindia, and eastasia will be released in January after the holidays as the next group of regions. We will update the release notes will the upcoming schedule going forward until all regions are deployed.
+* Kubernetes 1.19 will be removed on 2021-01-31.
```
Comment (Kaarthis) [inline CHANGELOG.md:12]: Just to play it safe and not track it back later , should we call out 1.19 will be removed in Q1 ? or are we completely confident about 1/31?

**Review by qpetraroia — APPROVED** [review]

**Review by Tatsinnit — APPROVED** [review]
🍾

File: CHANGELOG.md (hunk 3 above)
Comment (palma21) [reply CHANGELOG.md:12]: We are completely confident. And for removal dates it must be explicit/accurate.

**Pull request merged**